tensorflow/tools/docker/Dockerfile.devel-cpu-mkl

```diff
@@ -0,0 +1,96 @@
+FROM tensorflow/tensorflow:latest-devel
+
+LABEL maintainer="Clayne Robison<clayne.b.robison@intel.com>"
+
+# These arguments are parameterized. Use --build-args to override.
+ARG TF_BRANCH=r1.4
+ARG BAZEL_VERSION=0.7.0
+ARG WHL_DIR=/whl
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        golang \
+        vim \
+        emacs \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN pip --no-cache-dir install --upgrade \
+        pip setuptools
+
+RUN pip --no-cache-dir install wheel 
+
+# Use the specified Bazel version
+WORKDIR /
+RUN cd /bazel && \
```

Comment:
tensorflow:latest-devel should already have bazel installed.
Do we need to upgrade bazel for MKL?

Reply:
I thought that we had problems building MKL with it, but I just verified that it's OK. I'll remove this and use the version that comes with the base tensorflow container.